modules/infra/src/main/scala/scaladex/infra/GithubClient.scala

```diff
@@ -187,6 +189,18 @@ class GithubClient(token: Secret)(implicit val system: ActorSystem)
       .fallbackTo(Future.successful(Seq.empty))
   }
 
+  def getPercentageOfLanguage(ref: Project.Reference, language: String): Future[Int] = {
+    val request =
+      HttpRequest(uri = s"${repoUrl(ref)}/languages").addHeader(acceptJson).addCredentials(credentials)
+    for {
+      response <- get[Map[String, Int]](request)
+      languages = response.transform((_, numBytes) => numBytes.toFloat)
```

Comment:
Slightly concerned about using `.toFloat`, overflow might seem possible. Any thoughts on which datatype is preferred?

Reply:
`.toFloat` can lose precision on big int values but I don't think it can overflow so it seems fine.

But maybe we don't need to convert all numBytes values to float, only the one of the language:
```scala
for (response <- get[Map[String, Int]](request))
yield response.get(language)
  .fold(0)(bytesLang => ((bytesLang.toFloat / response.sum) * 100).toInt)
```